jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -81,13 +104,15 @@ def shutdown(self, msg):
                 return
 
         rospy.loginfo('Shut down robot.')
+        self.speak('シャットダウンします。')
```

Comment:
Is there any possibility that non-Japanese speakers will use this program? How about making this a parameter as well?

Reply:
I added parameters for non-Japanese and updated the document.
I tried it on real PR2 robot.